libcflib/whooshutils.py

```diff
@@ -0,0 +1,51 @@
+import os.path
+from whoosh.index import open_dir, exists_in, create_in
+from whoosh.fields import (Schema, TEXT, KEYWORD, BOOLEAN, NUMERIC, STORED,
+                           ID)
+
+
+SCHEMA = Schema(name=ID(stored=True),
```

Comment:
Also would it be of interest to put this inside the actual schema? Since we are using actual python we could put this in and write a validator extension which ignores this.